docs/help/getting-started/api-platform.md

```diff
@@ -0,0 +1,25 @@
+# Deploying docs from API Platform
+
+[API Platform](https://api-platform.com/) is an API-first PHP framework that allows you to create REST and GraphQL APIs using PHP classes, and automatically generates their documentation, based on the OpenAPI specification. If you are new to API Platform, check out [their getting started guide](https://api-platform.com/docs/distribution/).
```

Comment:
Shall we mention GraphQL?

Reply:
Good question. This is straight up coming from the API Platform description on their site.